tac/aea/mail/oef.py

```diff
@@ -327,6 +330,14 @@ def send_fipa_message(self, envelope: Envelope) -> None:
         else:
             raise ValueError("OEF FIPA message not recognized.")
 
+    def send_bytes(self, envelope: Envelope):
+        """Send a 'bytes' message."""
+        self.send_message(0, 0, envelope.to, envelope.encode())
```

Comment:
these 0s should be made constants so they can be referenced throughout and to aid clarity. E.g. `STUB_MESSSAGE_ID` & `STUB_DIALOGUE_ID` (then it is clear they are not used; see comment above also)